packages/pocket-ic/tests/tests.rs

```diff
@@ -1557,3 +1558,49 @@ fn subnet_metrics() {
     assert_eq!(metrics.num_canisters, 1);
     assert!((1 << 16) < metrics.canister_state_bytes && metrics.canister_state_bytes < (1 << 17));
 }
+
+#[test]
+fn test_raw_gateway() {
+    // We create a PocketIC instance consisting of the NNS and one application subnet.
+    let mut pic = PocketIcBuilder::new()
+        .with_nns_subnet()
+        .with_application_subnet()
```

Comment:
Two subnets because we need a root subnet, or whats the reason?

Reply:
Indeed, we need a root subnet for the HTTP gateway to function (the gateway uses an agent which requires a root key).